ethereum/contracts/Scale.sol

```diff
@@ -40,6 +40,44 @@ contract Scale {
         return number;
     }
 
+      // Convert bytes (big endian) to little endian format
+    function bytesToLittleEndian(bytes memory arr)
+        internal
+        pure
+        returns (uint256)
+    {
+        uint256 number;
+        for(uint i = 0; i < arr.length; i++){
```

Comment:
Again, to save gas, we could just iterate backwards over the array instead of reversing it, right?

Reply:
Created https://github.com/Snowfork/polkadot-ethereum/issues/56 for this. Small, but want to get this milestone functional and submitted before addressing some of these smaller optimizations.